### ISSUE TITLE
Auto-close after update

### ISSUE THREAD

**Issue by nightzirch** [gw2-addon-loader/GW2-Addon-Manager#162]
**Is your feature request related to a problem? Please describe.**
Not really a problem. Currently, we have the option to open GW2 on close. It would be nice to have another checkbox to auto-close the Addon Manager once the update is complete, resulting in one less click.

**Describe the solution you'd like**
1. I open GW2 Addon Manager
2. I click "Update"
3. The application is checking for updates
4. Optionally, the application is updating
5. The application automatically closes
6. GW2 is starting

**Describe alternatives you've considered**
I have considered being okay with having to click one more time than what could be possible.

**Additional context**
I made a fast mockup, but logically the two checkboxes should switch place.
![GW2-UAOAM-mockup](https://user-images.githubusercontent.com/1052787/141368021-a3ca0b37-1a77-4366-a2d8-1cd66a412d6c.jpg)
